### DISCUSSION TITLE
Latest version doesnt supporting dynamically create editor in a for loop angular

### DISCUSSION THREAD

**Discussion by gsaravanakumar932** [sibiraj-s/ngx-editor#410] [Q&A] [answered]
How to use multiple editor on same page ?

```ts
class AppComponent {
  editor: Editor;

  ngOnInit(): void {
    // this cant create if your using for loop, without creating instance of this Editor, it wont work
    this.editor = new Editor();
  }
}
```

```html
<div class="NgxEditor__Wrapper">
  <ngx-editor-menu [editor]="editor"> </ngx-editor-menu>
  <ngx-editor
    [editor]="editor"
    [ngModel]="html"
    [disabled]="false"
    [placeholder]="'Type here...'"
  ></ngx-editor>
</div>
```

any idea of how to use in for loop ?

**Comment by sibiraj-s** [comment, marked as answer]
You can create a component and intitalize the editor inside it, then loop the component instead. rather than initializing the editor on loop

**Reply by gsaravanakumar932** [reply]
@sibiraj-s  the name of the instance should be same as that of property in the [editor]="editor". I believe this cannot create dynamically.

eg: instance1 = new Editor;
      instance2 = new Editor;
       
[editor]="instance1 "
[editor]="instance2 "

like so.

**Reply by sibiraj-s** [reply]
Hence I suggested you to put the editor in a component and loop the component instead. So you don't have to worry about naming the instances. The editor will the self contained to the component. You just manipulate the components with inputs/data.

**Reply by gsaravanakumar932** [reply]
got it. Thanks... Working now.

[a comment by an automated account is not shown]
